registry/auth.go

```diff
@@ -164,26 +165,32 @@ func SaveConfig(configFile *ConfigFile) error {
 }
 
 // try to register/login to the registry server
-func Login(authConfig *AuthConfig, factory *utils.HTTPRequestFactory) (string, error) {
+func Login(authConfig *AuthConfig, factory *utils.HTTPRequestFactory, secure bool) (string, error) {
```

Comment:
as an FYI, the reason it was called `secure` is because it is both controlling SSL verification, but was also allowing HTTP fallback in other parts of the code.

@icecrime @dmp42 @shin- Not sure what's expected here wrt HTTP fallback for login.